README.md

```diff
@@ -167,36 +169,48 @@ tasker -tz America/New_York -file path/to/taskfile -shell zsh # run all tasks us
 > Same timezone applies for all tasks currently and it might support overriding timezone per task in future release.
 
 #### Notes on Windows
+
 In Windows if it doesn't find `bash.exe` or `git-bash.exe` it will use `powershell`.
 `powershell` may not be compatible with Unix flavored commands. Also to note:
 you can't do chaining with `cmd1 && cmd2` but rather `cmd1 ; cmd2`.
 
 ---
 ### Cron Expression
 
-Cron expression normally consists of 5 segments viz:
+A complete cron expression consists of 7 segments viz:
+```
+<second> <minute> <hour> <day> <month> <weekday> <year>
+```
+
+However only 5 will do and this is most commonly used. 5 segments are interpreted as:
 ```
 <minute> <hour> <day> <month> <weekday>
 ```
-and sometimes there can be 6th segment for `<year>` at the end.
+in which case a default value of 0 is prepended for `<second>` position.
 
-For each segments you can have multiple choices separated by comma:
-> Eg: `0,30 * * * *` means either 0th or 30th minute.
+In a 6 segments expression, if 6th segment matches `<year>` (i.e 4 digits at least) it will be interpreted as:
+```
+<minute> <hour> <day> <month> <weekday> <year>
```

Comment:
in 6 segments `year` is not included and starts with `second`

Reply:
this is for backward compat - in case 6th segment looks like year

Reply:
I got that but this will stick here forever. Maybe we can mention this as breaking change??? WDYT?